public/emoji/food.css

```diff
@@ -0,0 +1,853 @@
+/* glue: 0.13 hash: 31946c9658 */
+        .em-food-2615,
```

Comment:
and so on...

Reply:
This is a generated file :slightly_smiling_face: